metaflow/plugins/kubernetes/kubernetes_cli.py

```diff
@@ -107,6 +107,7 @@ def kubernetes():
     type=JSONTypeClass(),
     multiple=False,
 )
+@click.option("--shared-memory", default=None, help="Size of shared memory in MB")
```

Comment:
I believe it's not MB but MiB. We should look into other values like disk, tmpfs, memory and ensure that all fields have the same dimension

Reply:
Fixed.